### PR TITLE
chore: visual refresh on skeleton component

### PR DESCRIPTION
**Pull request by khamudom**
# Pull Request

## 📖 Description

before
image

after
image

### General

- [ ] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

### CODE REVIEW

File: packages/web-components/fast-components/src/skeleton/skeleton.styles.ts [hunk 1]
```diff
@@ -21,41 +25,36 @@ export const skeletonStyles: (
...
             --skeleton-animation-gradient-default: linear-gradient(
                 270deg,
                 var(--skeleton-fill, var(--skeleton-fill-default)) 0%,
-                #f3f2f1 51.13%,
+                ${neutralFillActive} 51.13%,
                 var(--skeleton-fill, var(--skeleton-fill-default)) 100%
             );
             --skeleton-animation-timing-default: ease-in-out;
         }
-
```
Comment (radium-v) [inline packages/web-components/fast-components/src/skeleton/skeleton.styles.ts:37]: Newlines between selector blocks help with readability.

**Review by robarbms — APPROVED** [review]
You need a change file, but otherwise looks good.

File: packages/web-components/fast-components/src/skeleton/skeleton.styles.ts (hunk 1 above)
Comment (khamudom) [reply packages/web-components/fast-components/src/skeleton/skeleton.styles.ts]: updated

**Comment by bheston** [comment]
I think this is probably fine, but the test fixture is off. Can you make the fixture use a Card or otherwise set the background color into dark mode (fillColor) so we can see that the skeleton colors are correct?

File: packages/web-components/fast-components/src/skeleton/fixtures/base.html [hunk 2]
```diff
@@ -1,32 +1,23 @@
...
+            background-color: buttontext;
         }
     }
 </style>
 
 <h1>Skeleton</h1>
 
-<h2>Used as element blocks</h2>
-<div class="card-placeholder">
+<h4>Used as element blocks</h4>
```
Comment (litteredwitherrors) [inline packages/web-components/fast-components/src/skeleton/fixtures/base.html:19]: What did we opt for heading level 4 here and below?

Comment (khamudom) [reply packages/web-components/fast-components/src/skeleton/fixtures/base.html:19]: that should stay h2

**Review by litteredwitherrors — APPROVED** [review]

**Pull request merged**